pandas/tests/frame/test_missing.py

```diff
@@ -602,9 +602,15 @@ def test_fillna_dict_series(self):
         expected = df.fillna(df.max().to_dict())
         tm.assert_frame_equal(result, expected)
 
-        # disable this for now
-        with pytest.raises(NotImplementedError, match="column by column"):
-            df.fillna(df.max(1), axis=1)
+        expected = DataFrame(
```

Comment:
make a new test

Reply:
@jreback 
I'm confused. you mean
1. just delete those codes and make a new test
2. change this tests as a new test